resources/lang/de.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Why do you want to mix 'short keys' and 'translation strings as keys'? I think staying with short keys is better, as we don't have to change all language files if we only wan't to fix a spelling mistake in the primary language.

Reply:
You're right, but Laravel uses this style for mails and much other content.

Reply:
So I can't change a lot here!